lib/sprig/seed/attribute.rb

```diff
@@ -54,7 +54,7 @@ def computed_value?(value)
       end
 
       def computed_value_regex
-        /<%[=]?(.*)%>/
+        /(<%[=]?(.*?)%>)/
```

Comment:
The extra `?` is a modifier that makes the match non-greedy. Without the ?, it would match the entirety of `<%= "Partially" %> <%= "Dynamic" %>` as a single match.